CMakeLists.txt

```diff
@@ -268,6 +269,22 @@ if (NOT ${TESTS_TO_RUN} STREQUAL "")
     )
 endif()
 
+if (${BUILD_PYTHON} AND ${RUN_PYTHON_TEST})
+    if (MSVC)
+        add_custom_command(
+            TARGET build-tests
+            POST_BUILD
+            COMMAND cp ${PYTHON_API_DIR}/Release/* ${PYTHON_API_DIR}
+        )
+    endif()
+
+    add_custom_command(
+        TARGET build-tests
+        POST_BUILD
+        COMMAND ${PYTHON_EXECUTABLE} -m pytest ${PYTHON_API_DIR}/test
+    )
+endif()    
+
```

Comment:
Did you test this on windows and unix?  shouldn't the second command be in else statement?

Reply:
The second command runs the python api tests, which should be done for both windows and unix. Windows puts the MarabouCore shared library in a Release subdirectory, which needs to be copied to the maraboupy directory to be used. Once it's copied, the tests run the same way for all build types. I tested on my laptop, and travis was able to build correctly. The travis outputs show that the tests were run and passed after the build.

Reply:
oops